src/chia_log/handlers/condition_checkers/non_decreasing_plots.py

```diff
@@ -25,14 +28,16 @@ def check(self, obj: HarvesterActivityMessage) -> Optional[Event]:
 
         event = None
         if obj.total_plots_count < self._max_farmed_plots:
-            message = (
-                f"Disconnected HDD? The total plot count decreased from "
-                f"{self._max_farmed_plots} to {obj.total_plots_count}."
-            )
-            logging.warning(message)
-            event = Event(
-                type=EventType.USER, priority=EventPriority.HIGH, service=EventService.HARVESTER, message=message
-            )
+            logging.info(f"Plots decreased from {self._max_farmed_plots} to {obj.total_plots_count}")
```

Comment:
Could be an if-else to avoid duplicate information (the same info is repeated, just with slightly different wording and priority)